app/components/searchable_dropdown_component.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class SearchableDropdownComponent < ViewComponent::Base
+  REMOVED_SEARCH_PLUGIN = {"tom-select-options-value": '{ "plugins": [] }'}.freeze
+  MINIMUM_OPTIONS_FOR_SEARCH_FILED = 11.freeze # at least 11 options are required for the search field
```

Comment:
Typo:
```suggestion
  MINIMUM_OPTIONS_FOR_SEARCH_FIELD = 11.freeze # at least 11 options are required for the search field
```

Reply:
I think this was handled in the further commits😅

Reply:
Actually, I just noticed in the diff that it's still `FILED`. Perhaps the fix got lost somewhere along the way (happens to me sometimes when I'm rebasing too much).

Reply:
Yeah I think I lost it somewhere while rebasing 😕

Thanks for the help here. ✨